analysers/experiments/cpu.py

```diff
@@ -59,39 +62,60 @@ def computeExperimentCoreMetrics(CassandraRDD, i):
         
         weightedMean = weightedSum/float(weightSum)
     
-    return {"median_min":medianMin, "median_max":medianMax, \
-              "min":dataMin, "max":dataMax, "q1_min":q1Min, \
+    return {"min":dataMin, "max":dataMax, "q1_min":q1Min, \
               "q1_max":q1Max, "q2_min":q2Min, "q2_max":q2Max, \
+              "p90_max":p90Max, "p90_min":p90Min, \
               "p95_max":p95Max, "p95_min":p95Min, \
+              "p99_max":p99Max, "p99_min":p99Min, \
               "q3_min":q3Min, "q3_max":q3Max, "weighted_avg":weightedMean}
  
-def createQuery(CassandraRDD, experimentID, containerID, hostID, nOfActiveCores):
-    from commons import computeExperimentMetrics, computeMetrics
+def createQuery(sc, cassandraKeyspace, srcTable, dataTable, experimentID, containerID, hostID):
+    from commons import computeExperimentMetrics, computeMetrics, computeLevene
+    
+    CassandraRDD = sc.cassandraTable(cassandraKeyspace, srcTable) \
+        .select("cpu_min", "cpu_max", "cpu_q1", "cpu_q2", "cpu_q3", "cpu_p90", "cpu_p95", "cpu_p99", "cpu_num_data_points", "cpu_mean", "cpu_me", "trial_id", "cpu_integral", "cpu_cores") \
+        .where("experiment_id=? AND container_id=? AND host_id=?", experimentID, containerID, hostID)
+    CassandraRDD.cache()
+    
+    CassandraRDDFirst = CassandraRDD.first()
```

Comment:
@VincenzoFerme the number of active cores I already compute at trial level, so at exp level I can take it from the trial table